### DISCUSSION TITLE
配置界面，右下角的添加按钮会覆盖配置文件设置的按钮。

### DISCUSSION THREAD

**Discussion by BaseBlank** [libnyanpasu/clash-nyanpasu#1527] [General]
<img width="766" alt="Snipaste_2024-09-08_19-11-19" src="https://github.com/user-attachments/assets/b918de4b-5a6c-4cea-97a1-0bdfc52f260c">

当有6个配置文件的时候，铺满整个界面，可能会出现配置界面的右下角的添加按钮会覆盖配置文件设置的按钮，对应那个配置文件设置就没办法点击。
不过不是大问题，拉一下整体软件界面就好。

**Comment by greenhat616** [comment]
你看看能不能点这个：
![图片](https://github.com/user-attachments/assets/920408d8-6d9d-4cf8-b519-879b0e22add2)
这应该是 issue，方便追踪后期修复。
